src/polygon.h

```diff
@@ -348,4 +348,9 @@ Datum	spherepoly_add_point(PG_FUNCTION_ARGS);
  */
 Datum	spherepoly_add_points_finalize(PG_FUNCTION_ARGS);
 
+/*
+ * Checks whether a polygon is convex
+ */
+Datum spherepoly_is_convex(PG_FUNCTION_ARGS);
```

Comment:
Add spaces after `Datum` so that `spherepoly_is_convex` lines up vertically with the names of the functions above this.